Remotely.Tests.ps1

```diff
@@ -1,4 +1,51 @@
-﻿Describe "Add-Numbers" {
+﻿# Function to update the table that defines the machineconfig.csv 
+# based on test environment variables
+Function Update-ConfigContentTable 
+{
+    param(
+        [parameter(Mandatory=$true)]
+        [HashTable]$configContentTable
+        
+    )
+
+    if($global:AppveyorRemotelyUserName -or $global:AppveyorRemotelyPassword)
+    {
+        $computername = $configContentTable.ComputerName
+        if($computername -ieq 'localhost')
+        {
+            $computername = '127.0.0.1'
+        }
+        elseif ( $computername -ieq '.')
+        {
+            $computername = '::1'
+        }
+        
+        $configContentTable.ComputerName = $computername
+        $script:expectedRemotelyTarget = $computername
+    }
+
+    if($global:AppveyorRemotelyUserName)
+    {
+        $configContentTable['Username']=$global:AppveyorRemotelyUserName
+    }
+
+    if($global:AppveyorRemotelyPassword)
+    {
+        $configContentTable['Password']=$global:AppveyorRemotelyPassword
+    }
+
+    return $configContentTable
+}
+
+$script:expectedRemotelyTarget = 'localhost'
```

Comment:
Instead of setting a script variable. can we use the value from $configContent ?